CONTRIBUTING.md

```diff
@@ -20,14 +20,22 @@ In this section, we describe how WatchDog needs to be setup. It is important to
 ## Eclipse
 1. Download Eclipse.
 1. Install the PDE plugin (https://marketplace.eclipse.org/content/eclipse-pde-plug-development-environment)
-1. Import `Maven -> Existing Maven project` and select the watchdog root folder. Import all subprojects except IntelliJ.
+1. Import `Maven -> Existing Maven project` and select the watchdog root folder. Import all subprojects except the IntelliJ folder (or remove it later if you have imported it, it will give build errors).
 1. Optional: For Maven IDE support, you need [m2e](https://www.eclipse.org/m2e/download/) for IDE-supported POM.xml-file editing. As we are also using [Tycho](https://eclipse.org/tycho/), you need the tycho configuration connector in m2e's marketplace.
 1. Optional: If your run configuration complains about unrecognized options, make sure to remove all inclusions of `--add-modules=ALL-SYSTEM` in your VM arguments. These are added incorrectly by PDE and won't work on a JDK 1.8 or lower.
 1. Select the correct target platform:
   1. `Window -> Preferences -> Plug-in Development -> Target Platform`
   1. Select the target platform from `/watchdog/eclipse/platform/watchdog.eclipse.platform.target`
 1. Works.
 
+After your are done, you will see a relatively complicated structure of projects. In the following, we explain each of them:
```

Comment:
Rather than having a list here, I think it is better to add README.md to the subprojects.

Reply:
Let's keep the information in one place, so people don't have to open 6 different READMEs (that don't even exist yet). If it poses a problem, we can refactor later.